pkg/config/app/kuma-cp/config.go

```diff
@@ -30,29 +28,19 @@ var _ config.Config = &Defaults{}
 
 type Defaults struct {
 	// Default Mesh configuration in YAML that will be applied on first usage of Kuma CP
-	Mesh string `yaml:"mesh"`
+	SkipDefaultMeshCreation bool `yaml:"skipDefaultMeshCreation"`
 }
 
 func (d *Defaults) Sanitize() {
 }
 
+// MeshProto gets the default mesh, which is hardcoded
 func (d *Defaults) MeshProto() v1alpha1.Mesh {
-	mesh, err := d.parseMesh()
-	util_error.MustNot(err)
-	return mesh
-}
-
-func (d *Defaults) parseMesh() (v1alpha1.Mesh, error) {
-	mesh := v1alpha1.Mesh{}
-	if err := proto.FromYAML([]byte(d.Mesh), &mesh); err != nil {
-		return mesh, errors.Wrap(err, "Mesh is not valid")
-	}
-	return mesh, nil
+	return v1alpha1.Mesh{}
 }
```

Comment:
I'd remove this function altogether. Let's have `v1alpha1.Mesh{}` in `CreateDefaultMesh`

Reply:
It will also have to go in `addnamespaceReconciler` as well, is that still ok?